workloads/src/main/scala/com/ibm/sparktc/sparkbench/workload/ml/LogisticRegressionBenchmML.scala

```diff
@@ -0,0 +1,104 @@
+package com.ibm.sparktc.sparkbench.workload.ml
+
+import com.ibm.sparktc.sparkbench.utils.GeneralFunctions._
+import com.ibm.sparktc.sparkbench.workload.Workload
+import org.apache.spark.ml.linalg.Vectors
+import org.apache.spark.ml.classification.LogisticRegression
+import org.apache.spark.ml.evaluation.{BinaryClassificationEvaluator => BCE}
+import org.apache.spark.sql.{DataFrame, Row, SparkSession}
+import org.apache.spark.sql.types.{LongType, StringType, StructField, StructType}
+
+// ¯\_(ツ)_/¯
+// the logic for this workload came from:
+// https://github.com/szilard/benchm-ml/blob/master/1-linear/5-spark.txt
+// ¯\_(ツ)_/¯
+
+case class LogisticRegressionBenchmML(
```

Comment:
I mainly chose it bc it uses a well-known, real-world dataset. All of the benchmarks look pretty straight forward too.